test/chemistry/mass_of_film_deposited_during_electrolysis_test.py

```diff
@@ -0,0 +1,66 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors, prefixes)
+from symplyphysics.laws.chemistry import mass_of_film_deposited_during_electrolysis as mass_law
+
+# Description
+## The current is 10 milliampere. The molar mass of a metal atom is 63.546 gram per mole. The current output is 1.05, and the valence of the metal atom is 29.
```

Comment:
How can be current output higher than 1? It is higher than input current?

Reply:
> How can be current output higher than 1? It is higher than input current?

The current output characterizes only the electrochemical process, i.e., for example, with the anodic dissolution of copper in a sulfuric acid electrolyte, the current output is close to 100%, but another 5% can be added due to the chemical dissolution of copper in the electrolyte. As a result, the calculated current output can formally be 105% due to the chemical dissolution of copper.

Reply:
Add some explanation to the description, to avoid future users' confusion.